publish.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import boto3
+from botocore.exceptions import ClientError
+
+
+def upload_file(file_name, bucket, object_name=None, s3_client=None):
+    """Upload a file to an S3 bucket
+
+    Arguments:
+    file_name (string): File to upload
+    bucket (string): Bucket to upload to
+    object_name (string): S3 object name. If not specified then file_name is used
+    s3_client (boto3.session.Session.client): S3 client to use. If not specified one is created. This is only useful for testing.
+
+    Returns:
+    boolean: True if file was uploaded, else False
+    """
+
+    # If no S3 client was specified, create a new one.
+    if s3_client is None:
+        s3_client = boto3.client('s3')
```

Comment:
**COMMENT:** The problem with that is we can only use credentials from environment variables.

Reply:
I wrote this guessing this is what we would do. Do you see it any other way in the near future? Unless it's clear we will use it in other ways, I would leave it like this for now. It can be refactored later anyway.